### DISCUSSION TITLE
https://airsdk.harman.com/ giving 504 Gateway Time-out

### DISCUSSION THREAD

**Discussion by jdarch** [airsdk/Adobe-Runtime-Support#1374] [General]
Hello. Apologies if this is not the correct place to raise this.

https://airsdk.harman.com/ has been down for several hours (it returns a 504 Gateway Time-out error message).

Thanks.

**Comment by ajwfrost** [comment]
Thanks, yes we're aware, it looks like an internal resource conflict -> one of the team is looking to sort it out and re-start it.

thanks

**Reply by ajwfrost** [reply]
Actually that was a red herring, it seems that the security rules were edited and AWS has a habit of dropping the ability to access ipv4 port 80 whenever a new rule is added... sorted now and hopefully next time it happens we'll think to immediately double-check those rules!

**Reply by jdarch** [reply]
Thanks for fixing it.

👍 

All the best.
